RapidA11y/UIViewController+Rapid.swift

```diff
@@ -34,8 +34,11 @@ extension UIViewController: RapidIdentifiable {
             return
         }
 
-        _ = controls.enumerated().map { index, control in
-            control.applyProperties(index, classOverride: controlsInformation[index].type)
+        _ = controls.enumerated().map {
+            let index = $0.offset
```

Comment:
can you restore the `index, control in` syntax, or was that not working? It's actually a lot slower to compile with the `$0`, and still fine here we are getting them out so might as well define them in the compile-friendly syntax :).